stable/msoms/templates/omsagent-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
```

Comment:
I would make it not install the daemonset if the values of the key and the wsid are default. Eg .https://github.com/samisms/charts/blob/master/stable/gcloud-sqlproxy/templates/deployment.yaml#L1

Reply:
Done